deepchem/splits/splitters.py

```diff
@@ -1211,79 +1425,51 @@ def update_distances(self, last_selected, cur_distances, distance_matrix,
         cur_distances[i] = new_dist
 
 
-class SpecifiedSplitter(Splitter):
-  """
-  Class that splits data according to user specification.
-  """
-
-  def __init__(self, input_file, split_field):
-    """Provide input information for splits."""
-    raw_df = next(load_data([input_file], shard_size=None))
-    self.splits = raw_df[split_field].values
-
-  def split(self,
-            dataset,
-            seed=None,
-            frac_train=.8,
-            frac_valid=.1,
-            frac_test=.1,
-            log_every_n=1000):
-    """
-    Splits internal compounds into train/validation/test by user-specification.
-    """
-    train_inds, valid_inds, test_inds = [], [], []
-    for ind, split in enumerate(self.splits):
-      split = split.lower()
-      if split == "train":
-        train_inds.append(ind)
-      elif split in ["valid", "validation"]:
-        valid_inds.append(ind)
-      elif split == "test":
-        test_inds.append(ind)
-      else:
-        raise ValueError("Missing required split information.")
-    return train_inds, valid_inds, test_inds
-
-
-class SpecifiedIndexSplitter(Splitter):
-  """
-  Class that splits data according to user index specification
-  """
-
-  def __init__(self, train_inds, valid_inds, test_inds):
-    """Provide input information for splits."""
-    self.train_inds = train_inds
-    self.valid_inds = valid_inds
-    self.test_inds = test_inds
+class TimeSplitterPDBbind(Splitter):
 
-  def split(self,
-            dataset,
-            seed=None,
-            frac_train=.8,
-            frac_valid=.1,
-            frac_test=.1,
-            log_every_n=1000):
+  def __init__(self, ids, year_file: Optional[str] = None):
     """
-    Splits internal compounds into train/validation/test by user-specification.
+    Parameters
+    ----------
+    ids: ....
+      WIP
+    year_file: str, optional (default None)
+      The file path of ....
```

Comment:
I could not understand what `ids` and `year_file` are

Reply:
I think here, `ids` are the PDB ids to be selected, and the `year_file` is the filepath for the PDBBind year selection. This is kind of arcane and it's not well supported right now.

Reply:
> This is kind of arcane and it's not well supported right now.

OK! I removed `TimeSplitterPDBbind ` from `__init__.py` and docs.